start-dconf-editor.sh

```diff
@@ -17,9 +19,17 @@ for dir in "${host_data_dirs[@]}"; do
     bridged=$(mktemp -d XXXXXXXXXX -p "$bridge_dir")
     mkdir -p "$bridged"/glib-2.0
     ln -s "$schemas" "$bridged"/glib-2.0
-    XDG_DATA_DIRS=$XDG_DATA_DIRS:"$bridged"
+    HOST_XDG_DATA_DIRS="${HOST_XDG_DATA_DIRS}:${bridged}"
   fi
 done
 
+# We MUST prepend the host's data dirs BEFORE the Flatpak environment's own dirs,
+# otherwise data (such as default values) load in the wrong order and would then
+# incorrectly prefer the Flatpak's internal defaults instead of the host's defaults!
+if [[ ! -z "${HOST_XDG_DATA_DIRS}" ]]; then
+  XDG_DATA_DIRS="${HOST_XDG_DATA_DIRS:1}:${XDG_DATA_DIRS}"
+fi
+
 export XDG_DATA_DIRS
+#echo "Using XDG_DATA_DIRS=${XDG_DATA_DIRS}"
```

Comment:
```suggestion
```

Reply:
@TingPing Fixed! I left that on purpose to make it easy for maintainers to debug by just uncommenting that, but I can agree that it's pointless. :)